lib/install.mjs

```diff
@@ -0,0 +1,132 @@
+import { createWriteStream, existsSync, rmSync, renameSync } from 'node:fs';
+import { get } from 'node:https';
+import { dirname, resolve } from 'node:path';
+import { arch, env, platform, exit } from 'node:process';
+import { fileURLToPath, URL } from 'node:url';
+
+import compressing from 'compressing';
+import progress from 'cli-progress';
+import semver from 'semver';
+
+import nodeManifest from "../package.json" assert { type: "json" };
+
+/**
+ * NW.js build flavor
+ * 
+ * @type {'sdk' | 'normal'}
+ */
+let buildType = env.npm_config_nwjs_build_type || env.NWJS_BUILD_TYPE || 'normal';
+
+// Parse Node manifest version.
+let parsedVersion = semver.parse(nodeManifest.version);
+let versionString = `${parsedVersion.major}.${parsedVersion.minor}.${parsedVersion.patch}`;
+
+// Check if version is a prelease.
+if (parsedVersion.prerelease !== undefined && typeof parsedVersion.prerelease[0] === 'string') {
```

Comment:
The original code only checked if the value was truthy. Specifying `!== undefined` infers to me that we are specifically treating this value differently if it is `null`, `false`, `0`, or `''`. However, from what I can tell, we seem to just be doing a safety check that it exists before checking if the first index is a string.

Since we are updating this code to modern JS standards anyway, we can simplify the truthy check even further with optional chaining.

```suggestion
if (typeof parsedVersion?.prerelease?.[0] === 'string') {
```

Reply:
👍🏼